pyartcd/pyartcd/oc.py

```diff
@@ -35,3 +39,47 @@ async def registry_login(runtime: Runtime):
     except ChildProcessError:
         runtime.logger.error('Failed to login into OC registry')
         raise
+
+
+def get_release_image_pullspec(release_pullspec: str, image: str):
```

Comment:
`get_release_image_pullspec` isn't clear to me as to what it does.. get release image from pullspec? get release image by pullspec? Also since we are running oc commands, I would suggest them prefixing them with `oc_` so it's clear what's happening and easier to find them. 
Maybe something like `oc_image_from_pullspec` ?

Reply:
this is to get the release image pullspec from the release pullspec

Reply:
the full name could be `get_release_impage_pullspec_from_release_pullspec` but I think it's too long